docs/setup.md

```diff
@@ -187,6 +187,23 @@ Alternatively you may submit an array of packages to define multiple versions:
 ]
 ```
 Once a custom package has been added to Private Packagist you will benefit from mirroring the zip file. Composer will have an additional location to download the file from if the original storage becomes unavailable.
+
+### Prevent a package from being used with custom packages
+
+While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used by creating a placeholder package. Setting its type to `metapackage` ensures no code is associated with it.
+
+```json 
+{
+  "name": "acme/blocked-package",
+  "version": "0.0.1",
+  "type": "metapackage"
```

Comment:
Maybe add a note about setting `abandoned` to `true` or a package name to tell users to not use the package and use a different package instead?

Reply:
Great idea, I've added that too!